hooks/993-factory-files.chroot

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# tmpfiles.d will copy files from /usr/share/factory So file that
+# needs to be populated in /writable have to be copied there.
+
+set -eu
+
+# cp -a requires /proc/self/fd in some cases
+mount -t proc proc /proc
+trap 'umount /proc' EXIT
+
+mkdir -p /usr/share/factory/writable/user-data
+cp -aT /home /usr/share/factory/writable/user-data/home
+
+dirs=(
```

Comment:
iiuc this is these are the auto/persistent/transition entries from the old `writable-paths` file, and I see two misses:
```
/etc/default/crda
/var/lib/apparmor
```
I see that actually those folders do not exist in UC20, so probably this is fine, but I'd like to understand what happened with them, if somebody knows. Especially with crda which is actually a file in classic.

Reply:
And same for `/etc/apparmor.d/cache`, I guess.